rust/arrow/src/ipc/mod.rs

```diff
@@ -36,3 +36,4 @@ pub use self::gen::SparseTensor::*;
 pub use self::gen::Tensor::*;
 
 static ARROW_MAGIC: [u8; 6] = [b'A', b'R', b'R', b'O', b'W', b'1'];
```

Comment:
is `const` better than `static` here? IIRC I used static for `ARROW_MAGIC` when const either wasn't a thing yet, or I was unfamiliar with it.

Reply:
Yea, I think this can be `const`.